packages/client/hmi-client/src/types/common.ts

```diff
@@ -201,6 +205,13 @@ export interface ChartSettingSensitivity extends ChartSettingBase, ChartSettingS
 	type: ChartSettingType.SENSITIVITY;
 }
 
+export interface ChartSettingComparison extends ChartSettingBase {
+	type: ChartSettingType.VARIABLE_COMPARISON;
+	smallMultiples?: boolean;
+	shareYAxis?: boolean;
+	showBeforeAfter?: boolean;
```

Comment:
Not super clear what `showBeforeAfter` means, can we add a comment here? or is there a better name for this toggle?

Reply:
Just added a comment with the explanation.